rfc021-platform-support-policy.md

```diff
@@ -65,12 +62,15 @@ Tier 2 supported platforms are those on which Omnitruck will serve packages, but
 * AIX 5.1L
 * FreeBSD 8
 * NetBSD
+* OpenBSD (i386 and amd64)
```

Comment:
https://github.com/chef/chef-rfc/pull/130
https://github.com/chef/omnibus-software/pull/426

The questions on OpenBSD are 1) where do platforms live that are working toward tier 2, tier 1.5? and 2) is this just waiting on Chef Software to either build the packages or provide a way for someone else to do so?